src/renderer/api/runtime.js

```diff
@@ -29,6 +30,8 @@ class Runtime {
   }
 
   getURL(path) {
+    const finalPath = path && path.startsWith('/') ? path : `/${path}`
```

Comment:
Is that used 🤔?

Reply:
Yes definitively, Gmelis use `runtime.getURL()` to have the extension URL, Mixmax use `runtime.getURL('/assets/my.asset)` and Grammarly use `runtime.getURL('assets/my.asset)` so this weird line fix make path canonical if needed.

Reply:
`finalPath` is not used.

Reply:
ahah fixed